receiver/iisreceiver/metadata.yaml

```diff
@@ -1,5 +1,13 @@
 name: iisreceiver
 
+resource_attributes:
+  iis.site:
+    description: The site of the web server.
+    type: string
+  iis.app_pool:
```

Comment:
I can't find any guidelines about abbreviated words, but I don't recall ever seeing any attributes that contain them. What do you think about changing this to `application_pool`?